### PR TITLE
Account comments, fix link in projects

### PR DESCRIPTION
**Pull request by victor11555**

### CODE REVIEW

File: packages/app/client/src/ui/pages/Account/components/MyProjects.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ function MyProjects(props) {
             key={project.id}
             onClick={() => {
               setCurrentProject({ project, userId })
-              history.push(`projects/${project.id}`)
+              history.push(`projects/${project.id}/about`)
```
Comment (veiga91) [inline packages/app/client/src/ui/pages/Account/components/MyProjects.js]: The route has changed. now is projects/view/:id/:tab

File: packages/app/client/src/ui/pages/Account/components/MyComments.js [hunk 2]
```diff
@@ -59,7 +72,15 @@ function MyComments(props) {
...
+        {comments.map((comment, i) => (
+          <CommentBox
+            removeComment={removeComment}
+            userIsTheOwner={comment.user.id === props.id}
+            key={comment.id}
+            author={comment.user.name}
+            {...comment}
+          />
+        ))}
         <Loader />
```
Comment (veiga91) [inline packages/app/client/src/ui/pages/Account/components/MyComments.js]: Are you still using this component? It is missing the isLoading prop

File: packages/app/client/src/ui/pages/Account/components/MyComments.js [hunk 3]
```diff
@@ -59,7 +72,15 @@ function MyComments(props) {
         headerText={`My comments (${comments.length})`}
         sortBy="date"
       >
-        <div>List</div>
+        {comments.map((comment, i) => (
```
Comment (veiga91) [inline packages/app/client/src/ui/pages/Account/components/MyComments.js]: You are not using the index, you can omit it.

File: packages/app/client/src/ui/pages/Account/components/MyComments.js [hunk 4]
```diff
@@ -49,7 +53,16 @@ function MyComments(props) {
...
+  const removeComment = handledRequest({
+    request: deleteComment,
+    showFullPageLoading: true,
+    onSuccess: () => {
+      setTrigger(!trigger)
+    },
+    successMessage: 'Comment successfully deleted'
+  })
 
   if (!Array.isArray(comments)) return null
```
Comment (veiga91) [inline packages/app/client/src/ui/pages/Account/components/MyComments.js]: Please add a way to handle the empty comments array

File: packages/app/client/src/ui/pages/Account/components/MyProjects.js [hunk 5]
```diff
@@ -63,28 +63,35 @@ function MyProjects(props) {
...
-        <Loader />
-      </SearchListWrapper>
-    </>
+    <SearchListWrapper
+      headerText={`My projects (${projects.length})`}
+      sortBy="date"
+    >
+      {projects.map(project => (
+        <ProjectCard
+          count={2}
```
Comment (veiga91) [inline packages/app/client/src/ui/pages/Account/components/MyProjects.js:83]: Are we still using this hard code?

**Review by veiga91 — APPROVED** [review]

**Review by NilsonBertolaAB — APPROVED** [review]

**Pull request merged**